OpenRA.Game/Graphics/Palette.cs

```diff
@@ -97,18 +97,18 @@ public void CopyToArray(Array destination, int destinationOffset)
 			Buffer.BlockCopy(colors, 0, destination, destinationOffset * 4, Palette.Size * 4);
 		}
 
-		public ImmutablePalette(string filename, int[] remap)
+		public ImmutablePalette(string filename, int[] remap, int background = 0, uint color = 0)
```

Comment:
I don't need this anymore with the fixed palette so I will just remove it. Saving to a local branch `configurable-palette-background` for later usage.